src/labels/CustomLabels.labels

```diff
@@ -1895,6 +1895,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <protected>true</protected>
         <shortDescription>The RD2 Batch job can only be executed when Enhanced RD is enabled</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can run the Enhanced Recurring Donations batch job.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_InstallmentFrequencyMustBeValid</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Validate Recurring Donations data if it has a valid installment frequency</shortDescription>
+        <value>Installment Frequency is required. Enter a value between 1 and 20.</value>
```

Comment:
@sam-knox Please review the new label

Reply:
@andrewyu-salesforce - Label looks good to me. Not sure about your question regarding a value of 0. I assumed that 0 was not a valid entry.